xarray/core/dataset.py

```diff
@@ -3934,9 +3935,23 @@ def merge(
             - 'left': use indexes from ``self``
             - 'right': use indexes from ``other``
             - 'exact': error instead of aligning non-equal indexes
+
         fill_value : scalar or dict-like, optional
             Value to use for newly missing values. If a dict-like, maps
             variable names (including coordinates) to fill values.
+        combine_attrs : {"drop", "identical", "no_conflicts", "drop_conflicts", \
+                        "override"}, default: "override"
+            String indicating how to combine attrs of the objects being merged:
```

Comment:
I guess this is `Dataset.attrs` not `Dataset[name].attrs`? We should mention that.

Reply:
I am planning to open another PR that adds support for merging `Dataset[name].attrs` (#4827 already added tests for that), so I think we can keep it as is.

Reply:
:+1:  thanks!